lib/post.php

```diff
@@ -253,7 +293,8 @@ public function meta() {
 			'post_date'    => $this->post->post_date,
 			'post_excerpt' => $this->post->post_excerpt,
 			'layout'       => get_post_type( $this->post ),
-			'permalink'    => get_permalink( $this->post )
+			'permalink'    => get_permalink( $this->post ),
+			'published'    => 'publish' === $this->status ? true : false,
```

Comment:
@mAAdhaTTah I believe there's a typo here:
```php
'published'    => 'publish' === $this->status ? true : false,
```
there's not `status` property, it's a method.

```php
'published'    => 'publish' === $this->status() ? true : false,
```

or Am I missing something?